ocaml/xapi/xapi_vusb.ml

```diff
@@ -34,6 +34,13 @@ let create ~__context ~vM ~uSB_group ~other_config =
         another vusb can not create with the same USB_group. *)
     if vusbs <> [] then
       raise (Api_errors.Server_error(Api_errors.usb_group_conflict, [Ref.string_of uSB_group]));
+
+    (* The host of the PUSB which will be attached to the vm must in possible_hosts that the vm can boot on.*)
```

Comment:
The code is fine but I think the error condition is not well captured in the comment or the error message. Make this as clear as possible.

Reply:
How about "All USBs that will be attached to the VM must be on the same host, as USB from host A can not attach to the vm resident on host  B. "